test/build.info

```diff
@@ -553,6 +549,10 @@ IF[{- !$disabled{tests} -}]
     INCLUDE[tls13encryptiontest]=.. ../include ../apps/include
     DEPEND[tls13encryptiontest]=../libcrypto ../libssl.a libtestutil.a
 
+    SOURCE[ideatest]=ideatest.c
+    INCLUDE[ideatest]=../include ../apps/include
+    DEPEND[ideatest]=../libcrypto.a libtestutil.a
```

Comment:
If we're going to treat this as an "internal" test - then shouldn't this be moved from the `PROGRAMS{noinst}` on line 30, to the one on line 503 for consistency with all the other internal tests?

Reply:
Good point, this will impact the other cipher deprecation PRs.

It either has to be made internal or removed -- I'm fine with either so long as we're consistent about it.  If it is removed, there won't be a direct low level test (but the APIs are deprecated).

Reply:
Done.